warehouse/templates/accounts/two-factor.html

```diff
@@ -89,7 +89,17 @@ <h2>{% trans %}Authenticate with an app{% endtrans %}</h2>
               <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
               {% endif %}
             </label>
-            {{ totp_form.totp_value(autocapitalize="off", autocomplete="off", inputmode="numeric", pattern="[0-9]*", required="required", spellcheck="false", class_="form-group__field", **{"aria-describedby":"totp-errors"}) }}
+            {{ totp_form.totp_value(
+              autocapitalize="off",
+              autocomplete="off",
+              inputmode="numeric",
+              pattern="[0-9]*",
+              required="required",
+              spellcheck="false",
+              class_="form-group__field",
+              aria_describedby="totp-errors",
```

Comment:
https://wtforms.readthedocs.io/en/2.3.x/widgets/#wtforms.widgets.html_params specifies that `aria_` params are rendered as `aria-` so no need for ** shenanigans.

Reply:
<img width="961" alt="Capture d’écran 2021-09-03 à 00 03 12" src="https://user-images.githubusercontent.com/1457576/131922218-cd4df223-edef-4105-8bfb-2c1bc842a681.png">
To dismiss any remaining doubt